doc/docs/concepts.md

```diff
@@ -212,3 +218,25 @@ restriction (always allow access), and then as a post-processing step adds the
 new Patient id to the client's patient access list. You can see this implemented
 in
 [`AccessGrantedAndUpdateList`](https://github.com/google/fhir-access-proxy/blob/main/plugins/src/main/java/com/google/fhir/gateway/plugin/AccessGrantedAndUpdateList.java).
+
+## AuditEvent logging
+
+Given that every access to the FHIR server goes through the Gateway, it's the
+ideal place to track access for auditing. The Gateway simplifies enabling
+AuditEvent logging with these key features:
+
+- Targeting the HL7 FHIR R4 specification -
```

Comment:
I think we can drop this as it is not really related to simplification and also it is kind of implied because of the next item (BALP). We actually have wanted to [support other FHIR versions](https://github.com/google/fhir-gateway/blob/1eaacd770dd5a098588eb302dd9ef3235ce88b72/server/src/main/java/com/google/fhir/gateway/FhirProxyServer.java#L60) for a long time.